documentation/commands/git-hack.md

```diff
@@ -13,9 +13,10 @@ git hack (--abort | --continue)
 
 #### DESCRIPTION
 
-Syncs the given parent branch,
+Syncs the given parent branch (default: main branch),
 forks a new feature branch with the given name off it,
-and brings over all uncommitted changes to the new feature branch.
+brings over all uncommitted changes to the new feature branch,
+and pushes everything to the remote repository.
```

Comment:
To me this makes it sound like it pushes up the previously uncommitted changes. The order here should probably be
```
Syncs the given parent branch (default: main branch),
forks a new feature branch with the given name off it,
pushes the new feature branch to the remote repository,
and brings over all uncommitted changes to the new feature branch.
```